single_level.py

```diff
@@ -16,35 +16,36 @@
 
 spreadsheet = os.path.join(config.get("Destinations", "directory"), config.get("Destinations", "filename"))
 
-#DACS Required Notes to check for
-required_notes = ["scopecontent", "accessrestrict"]
+def get_note_contents(resource, array, note_type):
```

Comment:
Can you make the indents in this function the same number of spaces as the other functions? Just a formatting thing that helps with human legibility!